bash_completion

```diff
@@ -25,8 +25,9 @@ __nvm_commands ()
     help install uninstall use run exec
     alias unalias reinstall-packages
     current list ls list-remote ls-remote
-    clear-cache deactivate unload
-    version which'
+    cache clear-cache deactivate unload
+    version version-remote which
+    debug'
```

Comment:
i kind of want `debug` not to autocomplete; there's no reason anybody should be using it except to fill out an issue.